tsconfig.json

```diff
@@ -1,20 +1,26 @@
 {
 	"compilerOptions": {
-		"target": "es5",
+		"target": "esnext",
+		"module": "esnext",
 		"lib": ["dom", "dom.iterable", "esnext"],
 		"allowJs": true,
 		"skipLibCheck": true,
 		"strict": true,
 		"forceConsistentCasingInFileNames": true,
 		"noEmit": true,
 		"esModuleInterop": true,
-		"module": "esnext",
 		"moduleResolution": "node",
 		"resolveJsonModule": true,
 		"isolatedModules": true,
 		"jsx": "preserve",
-		"incremental": true
+		"incremental": true,
+		"noImplicitReturns": true,
+		"noImplicitThis": true,
+		"noImplicitAny": true,
+		"strictNullChecks": true,
+		"noUnusedLocals": true,
+		"noUnusedParameters": true
 	},
-	"include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+	"include": ["next-env.d.ts", "src/**/*", "src/**/*", "*"],
```

Comment:
Can you revert this change? This was also generated by Next.js itself and is fine as it is.

Reply:
Hi @iShibi, thanks for taking the time to review this. I reverted this change but kept`"*"` to include config files in the root path, or the parser will raise an error. (or put them in ignore files)